### ISSUE TITLE
Not working anymore from last month

### ISSUE THREAD

**Issue by sentix001** [VovoloGames/WorkshopDL#39]
24 days ago I downloaded tons of mods for rimworld
via steamCMD/anon all worked fine

I came to download 3 more but after updating they're not downloading
both ANON and logged in

its just saying download failed in the console tries 3 times then says successfully downloaded and opens an empty folder

stderr log is empty

**Comment by sentix001** [comment]
not sure about previous version as I didnt use it then

I know it was 23 days as I had opened a ticket to name folders of mods instead of numbering after using it

**Comment by sean-permana** [comment]
Can confirm, it doesn't work anymore. Tried to download any Rimworld mods but it failed to download any mods.
Even if I downgrade the version, it still won't download any mods at all.

**Comment by sentix001** [comment]
![Screenshot - 11_20_2022 , 12_16_11 PM](https://user-images.githubusercontent.com/11886145/202899076-b5472a22-8211-45e4-9d4e-0e2d0e51d582.jpg)

**Comment by sentix001** [comment]
I was trying to download a list of 4 thats why it wasnt showing an error 
single mod it showed

**Comment by sean-permana** [comment]
Gotcha, I'll report back soon, guess it needs troubleshooting.

**Comment by sean-permana** [comment]
Yeah, just delete the entire workshopDL and install a new one, it works now. Also, remember to move all downloaded mods before deleting the whole workshopDL file. Thanks for the help!

**Comment by sentix001** [comment]
yes worked i only left the config file
and deleted rest

You need to mention this in the new version found header since all other programs that you update manually don't need the whole folder cleared up before updating so a lot of people will miss that
